src/System.Private.Uri/src/Resources/Strings.resx

```diff
@@ -258,4 +258,13 @@
   <data name="IO_PathTooLong" xml:space="preserve">
     <value>The specified file name or path is too long, or a component of the specified path is too long.</value>
   </data>
+  <data name="Argument_ExtraNotValid" xml:space="preserve">
+    <value>Extra portion of URI not valid</value>
+  </data>
+  <data name="Argument_InvalidUriPart" xml:space="preserve">
+    <value>Uri part is not is not valid.</value>
+  </data>
+  <data name="Argument_NotHexDigit" xml:space="preserve">
+    <value>Digit is not hex</value>
```

Comment:
"Digit is not hex" is not a complete sentence. Missing period.  Also, it is not proper English.

I would suggest "The character specified is not a valid hex digit."